### PR TITLE
completed challenge

### PR DESCRIPTION
**Pull request by kyllo34**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code-challenges/fizzBuzzTree/fizz-buzz-tree.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    this.left = left;
+    this.right = right;
+  }
+}
+
+function fizzBuzzTree(treeA) {
+  const rootA = treeA.root;
+  const treeB = new Tree(new Node);
+  const rootB = treeB.root;
+  function fizzBuzz(rootA, rootB) {
```
Comment (XiaojuanCai) [inline code-challenges/fizzBuzzTree/fizz-buzz-tree.js]: In most cases we do not write a function definition in another function definition. Better to pull it out.

File: code-challenges/fizzBuzzTree/fizz-buzz-tree.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      traverse(rootA.left, rootB.left)
+    }
+    if (rootA.right !== null) {
+      rootB.right = new Node();
+      traverse(rootA.right, rootB.right)
+    }
+  }
+  traverse(rootA, rootB);
+  return treeB;
+}
```
Comment (XiaojuanCai) [inline code-challenges/fizzBuzzTree/fizz-buzz-tree.js]: Excellent, your code generates a new fizzbuzz tree which is exactly the most challenging part of this code challenge.
